packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-phone/ios/integrationsteps.md

```diff
@@ -1,5 +1,366 @@
+## Integration steps
+
+### Summary
+
+There are many different ways to integrate the Swift SDK into your app. Using
+the sample code as a wrapper around the SDK, this guide describes the [steps](#steps)
+involved in integrating this use case into your application. Feel free to modify
+the sample code to fit your individual needs. The diagram below illustrates
+how the sample code and SDK fit into the overall integration.
+
+<div class="common-image-format">
+
+ ![Diagram showing the integration flow of the sample app and Swift SDK](/img/oie-embedded-sdk/oie-embedded-sdk-swift-sample-code-overview.png
+ "Swift sample and sdk diagram")
+
+</div>
+
+### Steps
+
+#### Step 1: Initialize MultifactorLogin
+
+The first step is to initialize `MultifactorLogin` and pass in a
```

Comment:
This set of docs is starting to help me understand the beauty of OIE. So basically, the code steps are nearly identical on some many of the use cases, and this is precisely because of the flexible nature of the SDKs, which can just deal with whatever facotrs, etc. you set up in the policies. So it is making me think that we should:

1. Talk about that power more.
2. Look to amalgamate the docs into one somehow, to have one doc that talks about all the different factor uses cases, and how OIE is so flexible. Saying that, I seem to remember you saying that the iOS SDK is more flexible than some of the others, so I wonder what is possible here?

Reply:
The Swift SDK is the only one, so far, that is entirely dynamic and leans into the dynamic nature of OIE. The other SDKs will follow suit, so this is a bit of a pathfinder project.  Thank you for your comments about it, because that's pretty much what I was going for.

Ultimately, I feel like we need to differentiate our documentation between the Imperative SDKs and the Dynamic SDKs.  This can enable us to clearly talk about the dynamic / cyclical workflow in OIE and the dynamic SDKs, and talk about the patterns more so than the specific remediation types.

Reply:
Mike and I were thinking of the same idea. The Swift SDK doesn't follow the other SDK's prescriptive model, it is more aligned with OIE's dynamic API.

Reply:
Cool, this is all very positive and encouraging. Once the OIE rush has died down a bit, at least for a while, we should get together and talk about what the docs structure should evolve into, to help explain these advantages and move this forward.